pkg/secrets/flags.go

```diff
@@ -71,12 +73,25 @@ func (o *CLIOptions) Complete(censor *DynamicCensor) error {
 	return nil
 }
 
+func (o *CLIOptions) NewReadOnlyClient(censor *DynamicCensor) (ReadOnlyClient, error) {
+	return o.NewClient(censor)
+}
+
 func (o *CLIOptions) NewClient(censor *DynamicCensor) (Client, error) {
 	if o.BwUser != "" {
 		c, err := bitwarden.NewClient(o.BwUser, o.BwPassword, censor.AddSecrets)
 		if err != nil {
 			return nil, fmt.Errorf("Failed to get Bitwarden client: %w", err)
 		}
+		if o.BwDefaultOrganization != "" || len(o.BwDefaultCollections) != 0 {
```

Comment:
Any reason to not hardcode that here? Do we ever use something different?

Reply:
No real reason, https://github.com/openshift/ci-tools/pull/1779/commits/e0eb3465d05b23921dfa762ba43378bed08caae5 makes it always apply.